plugin/help_func.go

```diff
@@ -0,0 +1,22 @@
+package plugin
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/mitchellh/cli"
+)
+
+func helpFunc(app string, plugins map[string]string) cli.HelpFunc {
+	return func(commands map[string]cli.CommandFactory) string {
+		var buf bytes.Buffer
+		if len(plugins) > 0 {
+			buf.WriteString("\n\nAvailable plugins commands are:\n")
+		}
+
+		for name, _ := range plugins {
+			buf.WriteString(fmt.Sprintf("    %s\n", name))
```

Comment:
This just shows the name right? I'm guessing this is missing actual help output from the plugin commands so far?

Reply:
Just the names.
If we want the help text, we will require plugins to have either:
a. a man page we could parse, or
b. a special command to return short synopsis, e.g: `$ trellis-my-plugin --synopsis`, or
c. some way to register synopsis like #145 

However, I think this is an overkill at the monment. How about we revisit this when we have thrid party plugins?

Reply:
Yeah if that's the only downside then I think it's fine to skip for now as well 👍